include/core/jbackend.h

```diff
@@ -126,6 +126,13 @@ struct JBackend
 	JBackendType type;
 	JBackendComponent component;
 
+	/**
+	 * delete everything in the backend
+	 * \param[in] which namespace to delete
+	 * \param[out] how many "things" are deleted within the specified namespace
+	 */
+	gboolean (*backend_reset)(gpointer, GError**);
```

Comment:
Why is this not inside one of the unions?

Reply:
the intention was to have one reset function for all backend types